docs/examples/jetson/concurrency_and_dynamic_batching/trtis_model_repo_sample_1/peoplenet/config.pbtxt

```diff
@@ -0,0 +1,28 @@
+name: "peoplenet"
```

Comment:
Since both the model and their configs are identical, why not re-use a single config and add the instance_group / dyanmic batching section per requirement.

Reply:
This is done on purpose so that the user does not have to modify the config.

Reply:
I see. I guess for the sake of simplicity we can allow for redundancy